build.gradle

```diff
@@ -27,7 +27,6 @@ allprojects {
     repositories {
         gradlePluginPortal()
```

Comment:
Do we need to opt out of metadata here as well?
```suggestion
        gradlePluginPortal() { metadataSources { mavenPom(); ignoreGradleMetadataRedirection() } }
```

Reply:
I think an excavator just enforces this??

Reply:
hmm, this did not do what I expected https://github.com/palantir/gradle-jdks/pull/3